SPECS/perl-Module-Build/perl-Module-Build.spec

```
@@ -82,6 +95,9 @@ LANG=C TEST_SIGNATURE=1 MB_TEST_EXPERIMENTAL=1 ./Build test
 %{_mandir}/man3/*
 
 %changelog
+* Tue Aug 23 2022 Muhammad Falak <mwani@microsoft.com> - 0.4231-2
+- Add BR on `perl(ExtUtils::*)` & `cpan` to enable ptest
```

Nit-pick: I think we're closer to adding `perl(CPAN::*)` than plain `cpan`. It doesn't matter much, but since you already used `perl(ExtUtils::*)`, then why not?:)

```suggestion
- Add BR on `perl(ExtUtils::*)` & `perl(CPAN::*)` to enable ptest
```